### PR TITLE
Display name

### LINKED ISSUE #36: Using Display_Name for More than Stops
Why does [this line work](https://github.com/umts/stop-project/blob/d069029d4805774f603d676a841b286760f82298/app/views/users/edit.haml#L3) if the method `display_name` is [expecting a stop](https://github.com/umts/stop-project/blob/d069029d4805774f603d676a841b286760f82298/app/helpers/bus_stops_helper.rb#L16)?

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #36

### CODE REVIEW

**Comment by mboneil10** [comment]
Tried to do the `item_type` thing? Couldn't figure it out. Please help.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+
+  def display_name(item)
+    case item.class
```
Comment (dfaulken) [inline app/helpers/application_helper.rb]: Case statements evaluate with `===`, so you can do things like

```ruby
case some_variable
when String
  puts some_variable
when Integer
  some_variable.succ.to_s
#...
end
```

Comment (dfaulken) [reply app/helpers/application_helper.rb]: Without needing to say `some_variable.class`, that is.

Comment (mboneil10) [reply app/helpers/application_helper.rb]: Thanks, I think I've only ever written one once before. Not a real excuse, but thanks!! I'll make the change now.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+
+  def display_name(item)
+    case item
+    when User
+      item.name.present? ? item.name : 'New user'
```
Comment (dfaulken) [inline app/helpers/application_helper.rb]: How about `item.name || 'New user'`?

**Pull request merged**